packages/jupyter-ai/package.json

```diff
@@ -104,7 +107,7 @@
     "stylelint-config-standard": "~24.0.0",
     "stylelint-prettier": "^2.0.0",
     "ts-jest": "^29",
-    "typescript": "~4.3.0",
+    "typescript": "~4.9.0",
```

Comment:
Was this intended? Minor versions of TS can change a lot so might be worth adding to PR description.

Reply:
Yes, I needed this to build with JupyterLab `4.1.0b0` packages.